target/scripts/helpers/change-detection.sh

```diff
@@ -73,5 +73,5 @@ function _monitored_files_checksums
   do
     [[ -f "${FILE}" ]] && CHANGED_FILES+=("${FILE}")
   done
-  sha512sum -- "${CHANGED_FILES[@]}"
+  [[ ${#CHANGED_FILES[@]} -ne 0 ]] && sha512sum -- "${CHANGED_FILES[@]}"
```

Comment:
Can we have a `return 0` afterwards please?:) otherwise this function may return with exit code 1, which could be undesired if we want to add `set -eE` in the future:)

Reply:
I'll wrap it within an if statement.